src/fec_base.h

```diff
@@ -755,6 +769,75 @@ void FecCode<T>::decode_lagrange(
         output->set(i, S.get(i));
 }
 
+template <typename T>
+void FecCode<T>::decode_vec_lagrange(
+    vec::Vector<T>* output,
+    const std::vector<Properties>& props,
+    off_t offset,
+    vec::Vector<T>* fragments_ids,
+    vec::Vector<T>* words,
+    vec::Vector<T>* vx,
+    int vx_zero)
+{
+    if (this->fft == nullptr) {
+        throw LogicError("FEC base: FFT must be initialized");
+    }
+    // std::cout << "WORDS:"; words->dump();
+    int k = this->n_data; // number of fragments received
+
+    vec::Poly<T> A(this->gf, n);
+    vec::Poly<T> _A_fft(this->gf, n);
+    vec::Poly<T> N_p(this->gf, n);
+    vec::Poly<T> N_p_ifft(this->gf, n);
+    vec::Poly<T> S(this->gf, k);
+
+    // compute A(x) = prod_j(x-x_j)
+    A.zero();
+    A.set(0, 1);
+    for (int i = 0; i < k; i++) {
+        A.mul_to_x_plus_coef(this->gf->sub(0, vx->get(i)));
+    }
+    // std::cout << "A(x)="; A.dump();
```

Comment:
These can be removed no? It's dead code (I guess it was useful when you were developping/debugging, but now it's unused).